t/apicast-log.t

```diff
@@ -0,0 +1,93 @@
+use lib 't';
+use Test::APIcast::Blackbox 'no_plan';
+
+# Can't run twice because one of the test checks the contents of the cache, and
+# those change between runs (cache miss in first run, cache hit in second).
+repeat_each(1);
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: validate that request_id is present.
+--- configuration
+{
+  "services": [
+    {
+      "id": 42,
+      "backend_version":  1,
+      "backend_authentication_type": "service_token",
+      "backend_authentication_value": "token-value",
+      "proxy": {
+        "api_backend": "http://test:$TEST_NGINX_SERVER_PORT/",
+        "proxy_rules": [
+          { "pattern": "/", "http_method": "GET", "metric_system_name": "hits", "delta": 2 }
+        ]
+      }
+    }
+  ]
+}
+--- backend
+  location /transactions/authrep.xml {
+    content_by_lua_block {
+      ngx.exit(200)
+    }
+  }
+--- upstream
+  location / {
+     echo 'yay, api backend: $http_host';
+  }
+--- request
+GET /?user_key=value
+--- response_body env
+yay, api backend: test:$TEST_NGINX_SERVER_PORT
+--- error_code: 200
+--- error_log
+requestID=
```

Comment:
Would be good to check more parts of the response. At least to make it clear that the requestID is written at the end of the line.